src/app/pages/BibPage.jsx

```diff
@@ -213,25 +213,33 @@ export const BibPage = (
           {electronicResources.length ? <ElectronicResources electronicResources={electronicResources} id="electronic-resources"/> : null}
         </section>
 
-        {items && items.length && !isElectronicResources ?
-          <section style={{ marginTop: '20px' }} id="items-table">
-            <ItemsContainer
-              displayDateFilter={bib.hasItemDates}
-              key={bibId}
-              shortenItems={location.pathname.indexOf('all') !== -1}
-              items={items}
-              bibId={bibId}
-              itemPage={searchParams.get('itemPage')}
-              searchKeywords={searchKeywords}
-              holdings={newBibModel.holdings}
-              itemsAggregations={reducedItemsAggregations}
-              mappedItemsLabelToIds={mappedItemsLabelToIds}
-              numItemsTotal={numItemsTotal}
-              numItemsCurrent={numItemsCurrent}
-              checkForMoreItems={checkForMoreItems}
-            />
-          </section>
-          : null
+        {/* Display the items filter container component when:
+          1: there are items through the `numItemsTotal` property,
+          2: there are items and none of those items are electronic resources,
```

Comment:
i think this should say "there are items and they are not all electronic resources"